### PR TITLE
Update Valid Prefix Set configuration and Border Router TLV.

### PR DESCRIPTION
**Pull request by jwhui**

### CODE REVIEW

**Comment by mszczodrak** [comment]
LGTM :+1:

**Comment by librasungirl** [comment]
LGTM 👍

**Comment by luwannest** [comment]
LGTM

**Comment by abtink** [comment]
LGTM 👍

File: src/cli/README.md [hunk 1]
```diff
@@ -479,7 +480,7 @@ Add a valid prefix to the Network Data.
 * prf: Default Router Preference, which may be: 'high', 'med', or 'low'.
 
 ```bash
-> route add 2001:dead:beef:cafe::/64 pvsr 0
+> route add 2001:dead:beef:cafe::/64 s med
```
Comment (xiaom-GitHub) [inline src/cli/README.md]: Notice that in CLI function ProcessRouteAdd(), if added route prefix is stable, will not process prf parameter.

**Comment by xiaom-GitHub** [comment]
LGTM

File: src/cli/README.md (hunk 1 above)
Comment (jwhui) [inline src/cli/README.md]: Good catch.  Let me fix this.

**Pull request merged**